client/src/components/messenger/Conversations.js

```diff
@@ -205,21 +199,23 @@ const Conversations = ({ selectConversation, socket }) => {
                 <button className="options" title="create conversation" onClick={() => handleClickAdd()}>
                     <Plus/>
                 </button>
-                { dialogOpen ? <AddConversationDialog onClose={addConversation} token={token} /> : <></> }
+                { dialogOpen ? <AddConversationDialog onClose={addNewConversation} token={token} /> : <></> }
             </div>
             <SearchBox id="search_mobile"/>
             <div id="conversations">
-                {conversations?.filter(filterConversations).sort(sortConversations).map((chat, i) => (
-                    <ConversationItem
-                        name={chat.title}
-                        img={chat.imgUrl ?? "https://picsum.photos/400?id=" + chat.title}
-                        lastMsg={chat.lastMsg ?? ""}
-                        lastDate={ConversationsController.evaluateDate(chat.lastDate)}
-                        unread={chat.unread}
-                        key={i}
-                        onClick={() => selectConversation(chat.id, chat.title, chat.participants)}
-                    />
-                ))}
+                {
+                    useSelector((state) => state.conversations.conversations)?.filter(filterConversations)
+                        .sort(sortConversations).map((chat, i) => (
+                            <ConversationItem
+                                name={chat.title}
+                                img={chat.imgUrl}
```

Comment:
This should be reverted to `img={chat.imgUrl ?? "https://picsum.photos/400?id=" + chat.title}` so the appropriate image is shown as the chat image id imageUrl doesn't exist